src/pages/Mypage/Mypage.js

```diff
@@ -0,0 +1,127 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+export default function MyPage() {
+  const [guestData, setGuestData] = useState([]);
+  const [hostData, setHostData] = useState([]);
+
+  useEffect(() => {
+    getGuestData();
+  }, []);
+
+  useEffect(() => {
+    getHostData();
+  }, []);
+
+  const getGuestData = () => {
+    axios
+      .get('http://localhost:3002/data/GuestData.json')
+      .then(response => setGuestData(response.data));
+  };
+
+  const getHostData = () => {
+    axios
+      .get('http://localhost:3002/data/HostData.json')
```

Comment:
`http://localhost:3002` 는 생략 가능합니다. 포트 번호가 달라지면(ex. localhost:3000) 코드 전체를 바꿔야 하기 때문에 리뷰 참고해서 수정해주세요!
```suggestion
      .get('/data/GuestData.json')
      .then(response => setGuestData(response.data));
  };

  const getHostData = () => {
    axios
      .get('/data/HostData.json')
```

Reply:
수정했습니다!